distributed/tests/test_variable.py

```diff
@@ -213,7 +213,6 @@ def f(i):
 
     futures = c.map(f, range(15))
     results = await c.gather(futures)
-    assert all(r > NITERS * 0.8 for r in results)
```

Comment:
This test is a bit odd overall. 

I would actually suggest to simplify `f`
- I don't see the point of sleeping in there
- I would prefer not using randomness in a test
- Why do we need to submit a task, compute it and store the result in there?

Removing all of this, I would expect this `r > NITERS` to be more robust. I'm perfectly fine removing this, though. We're provoking a non-safe computation and are asserting on the result. No way this can work reliably. Having a stressful get/set pattern within the worker w/out any exception should be sufficient to test if things work, e.g.

```python
def f(i):
        with worker_client() as c:
            v = Variable("x", client=c)
            for _ in range(NITERS):
                val = v.get()
                val += 1
                v.set(val)

v = Variable("x")
await v.set(1)  # No need to incorporate stored futures in this test, is there?

futures = c.map(f, range(15))
results = await c.gather(futures)
final = await v.get()
assert final > 1  # Maybe even final > NITERS but that _may_ be flaky again
```

Reply:
It's a very odd test, I agree.  I don't mind the extra future submission though.  It's just more oddness.  It's nice that the thing does complete happily.  I'm inclined to leave it as is (also just in the interest of moving on) but I'm happy to make the changes if you feel even moderately strongly about them.

Reply:
I spoke with @fjetter about this offline.  He mentioned that this was nice-to-have and not critical.  Moving on in the interests of time.